add-on/panels/test/sidebar_test.js

```diff
@@ -191,165 +171,32 @@ describe("loop.conversation", function() {
         mozLoop: {},
         sdkDriver: {}
       });
-      roomStore = new loop.store.RoomStore(dispatcher, {
-        activeRoomStore: activeRoomStore,
-        constants: {}
-      });
-      remoteCursorStore = new loop.store.RemoteCursorStore(dispatcher, {
+
+      textChatStore = new loop.store.TextChatStore(dispatcher, {
         sdkDriver: {}
       });
-      conversationAppStore = new loop.store.ConversationAppStore({
-        activeRoomStore: activeRoomStore,
-        dispatcher: dispatcher,
-        feedbackPeriod: 42,
-        feedbackTimestamp: 42,
-        facebookEnabled: false
-      });
 
       loop.store.StoreMixin.register({
-        conversationAppStore: conversationAppStore
-      });
-
-      addRemoteCursorStub = sandbox.stub();
-      clickRemoteCursorStub = sandbox.stub();
-      LoopMochaUtils.stubLoopRequest({
-        AddRemoteCursorOverlay: addRemoteCursorStub,
-        ClickRemoteCursor: clickRemoteCursorStub
+        activeRoomStore: activeRoomStore,
+        textChatStore: textChatStore
       });
 
-      loop.config = {
-        tilesIframeUrl: null,
-        tilesSupportUrl: null
-      };
-
       sinon.stub(dispatcher, "dispatch");
     });
 
     afterEach(function() {
       ccView = undefined;
     });
 
-    // XXX akita-sidebar
-    it.skip("should request AddRemoteCursorOverlay when cursor position changes", function() {
-
-      mountTestComponent();
-      remoteCursorStore.setStoreState({
-        "remoteCursorPosition": {
-          "ratioX": 10,
-          "ratioY": 10
-        }
-      });
-
-      sinon.assert.calledOnce(addRemoteCursorStub);
-    });
-
-    // XXX akita-sidebar
-    it.skip("should NOT request AddRemoteCursorOverlay when cursor position DOES NOT changes", function() {
-
-      mountTestComponent();
-      remoteCursorStore.setStoreState({
-        "realVideoSize": {
-          "height": 400,
-          "width": 600
-        }
-      });
-
-      sinon.assert.notCalled(addRemoteCursorStub);
-    });
-
-    // XXX akita-sidebar
-    it.skip("should request ClickRemoteCursor when click event detected", function() {
-
-      mountTestComponent();
-      remoteCursorStore.setStoreState({
-        "remoteCursorClick": true
-      });
-
-      sinon.assert.calledOnce(clickRemoteCursorStub);
-    });
-
-    // XXX akita-sidebar
-    it.skip("should NOT request ClickRemoteCursor when reset click on store", function() {
-
-      mountTestComponent();
-      remoteCursorStore.setStoreState({
-        "remoteCursorClick": false
-      });
-
-      sinon.assert.notCalled(clickRemoteCursorStub);
-    });
-
-    // XXX akita-sidebar
-    it.skip("should display the RoomView for rooms", function() {
-      conversationAppStore.setStoreState({ windowType: "room" });
+    it("should display the DesktopSidebarView", function() {
```

Comment:
We should also add a test for "ROOM_STATE.GATHER" since I believe that can happen, even if its only for a very short time.

Now I think about it, its probably best to also change the GATHER state just to display nothing, rather than the text, to avoid any possible flicker on slower machines.